packages/office-ui-fabric-react/src/components/Slider/Slider.types.ts

```diff
@@ -123,6 +123,24 @@ export interface ISliderProps extends React.ClassAttributes<SliderBase> {
    * @defaultvalue false
    */
   originFromZero?: boolean;
+
+  /**
+
```

Comment:
What is with this extra new line all over the new props with their documentation?